Anastasiya Santalova/grid/SCSS/hw2_grid.scss

```diff
@@ -0,0 +1,317 @@
+@import "_variables";
```

Comment:
Didn't SASS fail during the compilation? As far as I'm concerned, you should still import files without using underscore even though filenames contain it.

Reply:
Hmm, it works... But, OK! :)